src/Internal/QueryM/Pools.purs

```diff
@@ -79,10 +80,11 @@ getValidatorHashDelegationsAndRewards skh = do
   stringRep :: String
   stringRep = scriptHashToBech32Unsafe "script" $ unwrap skh
 
+  sh :: ScriptHash
+  sh = unwrap skh
+
   byteHex :: String
-  byteHex =
-    byteArrayToHex <<< unwrap <<< scriptHashToBytes <<< unwrap $
-      skh
+  byteHex = byteArrayToHex $ unwrap $ toBytes sh
```

Comment:
Doesn't it work without specifying a type for a new variable?

Reply:
No, but I'm changing it to use a type annotation instead of a variable. Seems like some oddity with the type checker, I'll link a discussion in slack